dev/com.ibm.ws.wssecurity_fat.wsscxf/publish/servers/com.ibm.ws.wssecurity_fat.ssl/jvm.options

```diff
@@ -0,0 +1 @@
+-Dcom.ibm.ws.beta.edition=true
```

Comment:
Is this going to enable beta for all tests using this server configuration, including tests that do not test checkpoint?
If so then I think we need to scope setting the beta flag only for checkpoint related tests.  Otherwise we risk breaking something for non-beta and missing it in the other tests.